includes/namespace.php

```diff
@@ -80,6 +80,60 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) ||
+	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
+) {
+	/**
+	 * Displays an admin notice about why the plugin is unable to load.
+	 *
+	 * @return void
+	 */
+	function _print_missing_build_admin_notice() {
+		?>
+		<div class="notice notice-error">
+			<p>
+				<strong><?php esc_html_e( 'Web Stories plugin could not be initialized.', 'web-stories' ); ?></strong>
+			</p>
+			<p>
+				<?php
+					echo wp_kses_post(
+						sprintf(
+						/* translators: %s: build commands. */
+							__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
+							'<code>composer install &amp;&amp; npm install &amp;&amp; npm run build</code>'
+						)
+					);
+				?>
+			</p>
+		</div>
+		<?php
+	}
+
+	add_action( 'admin_notices', __NAMESPACE__ . '\_print_missing_build_admin_notice' );
+}
+
+// In CLI context, existence of the JS files is not required.
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) &&
+	( ( defined( 'WP_CLI' ) && WP_CLI ) || 'true' === getenv( 'CI' ) || 'cli' === PHP_SAPI )
+) {
+	$heading = __( 'Web Stories plugin could not be initialized.', 'web-stories' );
+	$body    = sprintf(
+		/* translators: %s: build commands. */
+		__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
```

Comment:
```suggestion
		esc_html__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
```

Reply:
Don't think that's needed in a CLI context? 🤔

Reply:
Yeah, if someone filters the translated value to try and inject html. Also vip go review will normally flag this.

Reply:
I mean, I can add it, but HTML is not output in a CLI anyway, so... 🤷